builder-18.toml

```diff
@@ -1,3 +1,5 @@
+description = "Base builder for Heroku-18 stack, based on ubuntu:18.04 base image"
```

Comment:
Should it say something about "Heroku supported buildpacks" or otherwise what buildpacks it contains?

Reply:
The buildpacks are listed in the metadata blob already. We could mention _something_, but it feels like a fine place to start.